data/seeds/001-trips.js

```diff
@@ -0,0 +1,40 @@
+
+exports.seed = function (knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('trips').del()
```

Comment:
Since you're letting the ids default currently, you might want to use `truncate` here instead of `del`.